src/main/c/src/SerialImp.c

```diff
@@ -262,7 +262,7 @@ JNIEXPORT void JNICALL RXTXPort(Initialize)(
 {
 #if defined(DEBUG) && defined(__linux__) && defined(UTS_RELEASE)
 	struct utsname name;
-	char message[80];
+//	char message[80];
```

Comment:
Always delicate to know whether to comment things out or remove them entirely. In these cases, I think you've made an OK call by leaving them in place in case a future developer needs help debugging something.